### DISCUSSION TITLE
Is there a list of command line arguments?

### DISCUSSION THREAD

**Discussion by bruening-bw** [OpenBoard-org/OpenBoard#1194] [Q&A]
I looked for it, but found nothing. All I found was “-lang en” mentioned by someone. That's cool because I can now easily take screenshots with English UI. What else is out there?

Maybe I'm not smart enough to find it. But the problem with not being smart is that you need someone else to help you realize it. So here I am.

Maybe the list could be included in the parameters section of the wiki. I would offer my help.

Thanks in advance

**Comment by letsfindaway** [comment]
Let me try to make a first draft. The list of OpenBoard specific arguments is complete, but there may be others influencing a specific subsystem.

## OpenBoard specific arguments

- `-v`, `-verbose`, `verbose`, `-log`, `log` all have the same effect: OpenBoard will create a log file in the `log` subdirectory below the user data directory.
- `-lang xx` Switch the UI language to `xx`. `xx` is a two-letter country code. If the specified language is not supported, OpenBoard uses English. 

## Arguments for the WebEngine

- `--single-process` Use a single process for all web views (web pages and widgets). This could reduce memory consumption on documents with many widgets. By default OpenBoard instructs the WebEngine to use one process per site.

## Windows specific arguments

- `-platform windows:nowmpointer` Used to mitigate a problem where dots are appearing at the beginning of a short stroke drawn with a pen.
